### ISSUE TITLE
Need ldapsearch installed

### ISSUE THREAD

**Issue by coretl** [dls-controls/dev-c7#49]
```
$ configure-tool e -p dls_scripts 4-3
/dls_sw/prod/tools/RHEL7-x86_64/defaults/bin/configure-tool: line 113: ldapsearch: command not found
Unable to determine e-mail address
```

**Comment by gilesknap** [comment]
we need to add 

yum install -y  openldap-clients

**Comment by gilesknap** [comment]
fixed in 2.0.4
